pyc64/basic.py

```diff
@@ -437,17 +437,15 @@ def execute_on_goto_gosub(self,cmd):
             else:
                 raise BasicError("syntax")
         targetLineList=targetLineList.strip()
-        # Arrayz...ize it        
+        # Make a tuple of the target list         
         lineTargetTuple=eval("("+targetLineList+")")
```

Comment:
The eval was added to be able to do  **on X goto**
which is why the on goto was invented :)

Also, I think the comment explain better the idea, so I'd like to leave it but feel free to remove it.

Reply:
well, if it's only used to turn a target line list like "20, 30"  into a tuple (20, 30)  I think a solution using split() and int()  is preferable to using eval(),  but that can't parse a target line list such as "20+10, 50/2" if you wanted to support that. Which I think is a bit silly hence my preference to rewriting this  (it's best practice to avoid using eval whenever possible)

Reply:
Yes, you are right. I will avoid using eval on that line (sorry I did not understand in the first place :-) 
We will use split() to get an array and then int() to the target element.
I too avoid eval if possible.